dpe/src/commands/derive_context.rs

```diff
@@ -1570,4 +1852,35 @@ mod tests {
             };
         }
     }
+
+    fn dpe_command_and_expected_child_count(
```

Comment:
nit: maybe a name `dpe_command*` feels a bit generic because it can only support DeriveContextCmd. Would a name like `derive_context_and_expected_child_count` be more applicable?

Reply:
I agree! I updated the names.